test/model.js

```diff
@@ -428,27 +421,24 @@
   });
 
   QUnit.test("defaults", function(assert) {
-    assert.expect(4);
+    assert.expect(5);
     var Defaulted = Backbone.Model.extend({
       defaults: {
-        "one": 1,
-        "two": 2
+        one: 1,
+        two: 2
       }
     });
     var model = new Defaulted({two: undefined});
     assert.equal(model.get('one'), 1);
     assert.equal(model.get('two'), 2);
+    model = new Defaulted({two: 3});
+    assert.equal(model.get('one'), 1);
+    assert.equal(model.get('two'), 3);
     Defaulted = Backbone.Model.extend({
-      defaults: function() {
-        return {
-          "one": 3,
-          "two": 4
-        };
-      }
+      defaults: { hasOwnProperty: true, }
     });
-    model = new Defaulted({two: undefined});
```

Comment:
> No... it's testing that passing undefined is thrown away, not that the key isn't passed at all.

Isn't that [this test](https://github.com/jridgewell/backbone/blob/defaults-prototype-collision/test/model.js#L425-L433)?

> `var defaults = {foo: true};`

With my "defaults" implementation:

```js
defaults = {foo: true};

defaults({}, defaults, {foo: undefined}); // { foo: true }
defaults({}, defaults, {bar: undefined}); // { foo: true, bar: undefined }
```

The only difference with mine is it won't respect properties that are already on the `dest` object:

```js
_.defaults({foo: false}, {foo: true}); // { foo: false }
defaults({foo: false}, {foo: true}); // { foo: true }
```